lib/cl/externals/software/alexandria-20170830-git/.boring

```diff
@@ -0,0 +1,13 @@
+# Boring file regexps:
```

Comment:
If you checked in alexandria, it should not be part of the submission.  Instead, the build process should install it on demand in the same manner that the nodejs implementation calls npm to install packages during the build.